flow/hooks/__init__.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (c) 2018 The Regents of the University of Michigan
```

Comment:
I agree with the idea that `__init__.py` should be clean if possible. If we don't have the other hooks from #189, this could just live in `hooks.py` in the root of the package (without a subdirectory). However, it would be good to separate hook implementations into their own files in the `hooks/` subdirectory.